l10n_ch_fds_postfinance/wizards/fds_key_import_wizard.py

```diff
@@ -0,0 +1,100 @@
+# © 2015 Compassion CH (Nicolas Tran)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api, exceptions, _
+import logging
+import base64
+
+_logger = logging.getLogger(__name__)
+
+
+class FdsKeyImportWizard(models.TransientModel):
+    """ FDS Postfinance keys import wizard.
+        The goal is to import existing key in the database.
+
+        This wizard is called when we click on import FDS authentication keys
+        for one FDS.
+        This Class inherit from fds_key_generator_wizard.
+    """
+    _name = 'fds.key.import.wizard'
+    _inherit = 'fds.key.generator.wizard'
+
+    public_key_import_txt = fields.Text(
+        string='Public key',
+        help='copy/paste your public key'
+    )
+    private_key_import_txt = fields.Text(
+        string='Private key',
+        help='copy/paste your private key'
+    )
+    public_key_import_file = fields.Binary(
+        string='Public key',
+        help='select one file that contain your public key'
+    )
+    private_key_import_file = fields.Binary(
+        string='Private key',
```

Comment:
Warning your `string` are the same for:
- public_key_import_txt and public_key_import_file
- private_key_import_txt and private_key_import_file